homework/index.js

```diff
@@ -16,31 +37,139 @@
     xhr.send();
   }
 
-  function createAndAppend(name, parent, options = {}) {
-    const elem = document.createElement(name);
-    parent.appendChild(elem);
-    Object.keys(options).forEach(key => {
-      const value = options[key];
-      if (key === 'text') {
-        elem.textContent = value;
-      } else {
-        elem.setAttribute(key, value);
+  function Contributions(inText) {
+    const url = inText.innerHTML;
+    const infoContainer = document.getElementById('info_container');
+    const infoRight = createAndAppend('div', infoContainer, {
+      class: 'info_right',
+      text: 'Contributions:',
+    });
+    fetchJSON(url, (err, data) => {
+      if (err) {
+        renderError(err);
+        return;
       }
+      data.forEach(element => {
+        const ContributorEl = createAndAppend('a', infoRight, {
+          class: 'Contributor_el',
+          href: element.html_url,
+          target: '_blank',
+        });
+        const imgDiv = createAndAppend('div', ContributorEl, { class: 'Contributor_img_div' });
+        createAndAppend('img', imgDiv, {
+          class: 'Contributor_img',
+          src: element.avatar_url,
+          alt: 'Contributor photo',
+          width: '50px',
+        });
+        const dataDiv = createAndAppend('div', ContributorEl, { class: 'Contributor_data_div' });
+        createAndAppend('div', dataDiv, {
+          class: 'Contributor_name',
+          text: element.login,
+        });
+        createAndAppend('div', dataDiv, {
+          class: 'Contributor_contributions',
+          text: element.contributions,
+        });
+        createAndAppend('hr', infoRight, {
+          class: 'hr_contributor',
+        });
+      });
+    });
+  }
+
+  function viewRepInfo() {
```

Comment:
If you want to abbreviate the word repository I would make it `repo` instead of `rep` . The word `repo` is often use in speech to mean repository. But nobody would conclude that you mean repository when you say the word `rep`.